tests/Phinx/Db/Adapter/SQLiteAdapterTest.php

```diff
@@ -1923,7 +1923,7 @@ public function testBulkInsertDates()
         $rows = $this->adapter->fetchAll('SELECT * FROM table1');
         $this->assertEquals('foo', $rows[0]['name']);
         $this->assertEquals('bar', $rows[1]['name']);
-        $this->assertMatchesRegularExpression('/[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}/', $rows[0]['created']);
+        $this->assertMatchesRegularExpression('/[0-9]{4}-[0-9]{2}-[0-9]{2}/', $rows[0]['created']);
```

Comment:
Any idea whats that about?

Reply:
Sqlite doesn't have types in the way other databases do, where they're more "suggestions" on what the data in the column is vs what it actually is. So while all other databases will coerce a date string (`2025-04-17`) into a proper datetime for the column type, sqlite leaves it as a date string.